libsrc/telestrat/bordercolor.s

```diff
@@ -0,0 +1,16 @@
+;
+; Ullrich von Bassewitz, 06.08.1998
+;
+; unsigned char __fastcall__ bordercolor (unsigned char color);
+;
+
+
+        .export         _bordercolor
+
+        .include        "telestrat.inc"
+
+_bordercolor:
+        ; Nothing to do
+        ; Oric can't handle his border
+        rts
```

Comment:
`bordercolor` needs to return a value! Checkout https://github.com/cc65/cc65/blob/master/libsrc/apple2/color.s#L16 on how to do that best.

Additionally check out https://github.com/cc65/cc65/blob/master/include/apple2.h#L203-L211 on how to further reduce size/cycles.

Reply:
Fixed let me know if my correction is good.